autoload/ale/toggle.vim

```diff
@@ -14,7 +14,9 @@ function! s:DisablePostamble() abort
         call ale#highlight#UpdateHighlights()
     endif
 
-    if g:ale_virtualtext_cursor isnot# 'disabled' && g:ale_virtualtext_cursor != 0
+    if g:ale_virtualtext_cursor isnot# 'disabled'
+    \&& g:ale_virtualtext_cursor isnot# 0
+    \&& g:ale_virtualtext_cursor isnot# '0'
```

Comment:
So I had a question about this `if` clause.  While this is definitely an edge case, if within a single session a user had `g:ale_virtualtext_cursor` enabled, then set it to `'disabled'`, and then ran `ALEDisable`, the virtual texts would not be cleared.  Would it be better to call `ale#virtualtext#Clear()` uncoditionally?